src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java

```diff
@@ -77,33 +86,41 @@ public static void rect(ResourceLocation texture, int left, int top, int width,
 	/**
 	 * Draws an untextured rectangle of the specified RGB color. Alpha is always 1.0f.
 	 */
-	public static void rect(int left, int top, int width, int height, int color) {
+	public static void drawRectangle(int left, int top, int width, int height, int color) {
 		if (width <= 0) width = 1;
 		if (height <= 0) height = 1;
 		
-		float r = (color >> 16 & 255) / 255.0F;
-		float g = (color >> 8 & 255) / 255.0F;
-		float b = (color & 255) / 255.0F;
+		float r = (color >> 16 & 255) / 255.0f;
+		float g = (color >> 8 & 255) / 255.0f;
+		float b = (color & 255) / 255.0f;
+		float a = 1.0f;
 		Tessellator tessellator = Tessellator.getInstance();
 		BufferBuilder buffer = tessellator.getBuffer();
 		GlStateManager.enableBlend();
 		GlStateManager.disableTexture2D();
 		GlStateManager.tryBlendFuncSeparate(GlStateManager.SourceFactor.SRC_ALPHA, GlStateManager.DestFactor.ONE_MINUS_SRC_ALPHA, GlStateManager.SourceFactor.ONE, GlStateManager.DestFactor.ZERO);
-		GlStateManager.color(r, g, b, 1.0f);
+		GlStateManager.color(r, g, b, a);
 		buffer.begin(GL11.GL_QUADS, DefaultVertexFormats.POSITION); //I thought GL_QUADS was deprecated but okay, sure.
-		buffer.pos(left,         top + height, 0.0D).endVertex();
-		buffer.pos(left + width, top + height, 0.0D).endVertex();
-		buffer.pos(left + width, top,          0.0D).endVertex();
-		buffer.pos(left,         top,          0.0D).endVertex();
+		buffer.pos(left,         top + height, 0).endVertex();
+		buffer.pos(left + width, top + height, 0).endVertex();
+		buffer.pos(left + width, top,          0).endVertex();
+		buffer.pos(left,         top,          0).endVertex();
 		tessellator.draw();
 		GlStateManager.enableTexture2D();
 		GlStateManager.disableBlend();
 	}
 
 	/**
-	 * Draws a rectangle for a Fluid, because fluids are tough.
+	 * Draws a rectangle for a fluid, because fluids are tough.
+	 */
+	public static void drawRectangle(Fluid fluid, int left, int top, int width, int height, float u1, float v1, float u2, float v2) {
+		drawRectangle(fluid, left, top, width, height, u1, v1, u2, v2, 0xFFFFFFFF);
```

Comment:
I'm not sure how I feel about an overload that defaults to white, as it hides some information for little reason. Rendering code is always painfully verbose and this does nothing to help that.